packages/frontend/component/src/components/setting-components/share.css.ts

```diff
@@ -115,31 +115,19 @@ globalStyle(`${storageProgressWrapper} .storage-progress-desc`, {
 globalStyle(`${storageProgressWrapper} .storage-progress-bar-wrapper`, {
   height: '8px',
   borderRadius: '4px',
-  backgroundColor: 'var(--affine-pure-black-10)',
+  backgroundColor: 'var(--affine-black-10)',
   overflow: 'hidden',
 });
 export const storageProgressBar = style({
   height: '100%',
   backgroundColor: 'var(--affine-processing-color)',
   selectors: {
-    '&.warning': {
```

Comment:
why this is deleted, I have seem this still exists in design file.

Reply:
I had a discussion with the designer @hwangdev97  about this colour and status, and this is the outcome of that discussion.